cmake/find/FindNASM.cmake

```diff
@@ -0,0 +1,35 @@
+# This module will set `NASM_FOUND` if NASM is found.
+# If NASM is found, `NASM_ROOT` will be set to the root folder.
+# `NASM_BIN_DIR` will be set to the folder containing the executable,
+# and `NASM_EXECUTABLE` will be set to the path of the NASM executable.
+
+
+if(HUNTER_STATUS_DEBUG)
+  message("[hunter] Custom FindNASM module")
+endif()
+
+# Try to find it even if it may already be set.
+find_path(
```

Comment:
Can you please explain what is happening here? Why we are not using `find_executable`?